src/ts/schemaDesigner/schemaDesignerEntity.ts

```diff
@@ -0,0 +1,84 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA.See License.txt in the project root for license information.
```

Comment:
I think this code is MIT now, at least ADS license was changed, so I think the source eula is gone now.  But if this is what all the code says, let's just leave it for now.